daemon/daemon_main.go

```diff
@@ -782,11 +787,13 @@ func initConfig() {
 
 	// If a config file is found, read it in.
 	if err := viper.ReadInConfig(); err == nil {
-		log.Infof("Using config file: %s", viper.ConfigFileUsed())
+		log.WithField(logfields.Path, viper.ConfigFileUsed()).
+			Info("Using config from file")
 	} else if option.Config.ConfigFile != "" {
-		log.Fatalf("Error reading config file %s: %s", option.Config.ConfigFile, err)
+		log.WithField(logfields.Path, option.Config.ConfigFile).
+			Fatal("Error reading config file")
 	} else {
-		log.Warnf("Error reading default config: %s", err)
+		log.WithField(logfields.Reason, err).Info("Skipped reading configuration file")
```

Comment:
I set this to use `logfields.Reason` just because the error isn't really something that users need to concern themselves with unless they're expecting to use this functionality. Here's how it will appear in the logs:

```
level=info msg="Skipped reading configuration file" reason="Config File \"ciliumd\" Not Found in \"[/home/vagrant]\"" subsys=daemon
```

It's really searching `ciliumd.[yaml|json|...]`, but apparently the upstream library's log message doesn't reveal this very well in its error message generation.